src/connector/src/parser/plain_parser.rs

```diff
@@ -113,11 +146,263 @@ impl ByteStreamSourceParser for PlainParser {
     }
 
     async fn parse_one<'a>(
+        &'a mut self,
+        _key: Option<Vec<u8>>,
+        _payload: Option<Vec<u8>>,
+        _writer: SourceStreamChunkRowWriter<'a>,
+    ) -> Result<()> {
+        unreachable!("should call `parse_one_with_txn` instead")
+    }
+
+    async fn parse_one_with_txn<'a>(
         &'a mut self,
         key: Option<Vec<u8>>,
         payload: Option<Vec<u8>>,
         writer: SourceStreamChunkRowWriter<'a>,
-    ) -> Result<()> {
+    ) -> Result<ParseResult> {
+        tracing::info!("parse_one_with_txn");
```

Comment:
Will this be too verbose?

Reply:
My bad, will remove it in a following bugfix PR.